src/jdk.management/share/classes/com/sun/management/ThreadMXBean.java

```diff
@@ -105,6 +105,41 @@ public interface ThreadMXBean extends java.lang.management.ThreadMXBean {
      */
     public long[] getThreadUserTime(long[] ids);
 
+    /**
+     * Returns an approximation of the total amount of memory, in bytes,
+     * allocated in heap memory since the Java virtual machine was launched,
+     * including the amount allocated by terminated threads.
```

Comment:
This "including ..." part seems redundant - it is the value allocated since JVM launch.

Reply:
Removed.